projects/rulenode-core-config/src/lib/components/action/azure-iot-hub-config.component.html

```diff
@@ -0,0 +1,102 @@
+        <ng-template ngSwitchCase="anonymous">
+        </ng-template>
+        <ng-template ngSwitchCase="sas">
+          <mat-form-field class="mat-block">
+            <mat-label translate>tb.rulenode.sas-key</mat-label>
+            <input type="password" required matInput formControlName="sasKey">
+            <mat-error *ngIf="azureIotHubConfigForm.get('credentials').get('sasKey').hasError('required')">
+              {{ 'tb.rulenode.sas-key-required' | translate }}
+            </mat-error>
+          </mat-form-field>
+          <tb-file-input formControlName="caCert"
+                         inputId="caCertSelect"
+                         [existingFileName]="azureIotHubConfigForm.get('credentials').get('caCertFileName').value"
+                         (fileNameChanged)="azureIotHubConfigForm.get('credentials').get('caCertFileName').setValue($event)"
+                         requiredAsError
+                         label="{{'tb.rulenode.azure-ca-cert' | translate}}"
+                         noFileText="tb.rulenode.no-file"
+                         dropLabel="{{'tb.rulenode.drop-file' | translate}}">
+          </tb-file-input>
+        </ng-template>
+        <ng-template ngSwitchCase="cert.PEM">
+          <tb-file-input formControlName="caCert"
+                         inputId="caCertSelect"
+                         [existingFileName]="azureIotHubConfigForm.get('credentials').get('caCertFileName').value"
+                         (fileNameChanged)="azureIotHubConfigForm.get('credentials').get('caCertFileName').setValue($event)"
+                         requiredAsError
+                         label="{{'tb.rulenode.azure-ca-cert' | translate}}"
+                         noFileText="tb.rulenode.no-file"
+                         dropLabel="{{'tb.rulenode.drop-file' | translate}}">
+          </tb-file-input>
+          <tb-file-input formControlName="cert"
+                         inputId="CertSelect"
+                         [existingFileName]="azureIotHubConfigForm.get('credentials').get('certFileName').value"
+                         (fileNameChanged)="azureIotHubConfigForm.get('credentials').get('certFileName').setValue($event)"
+                         required
+                         requiredAsError
+                         label="{{'tb.rulenode.cert' | translate}}"
+                         noFileText="tb.rulenode.no-file"
+                         dropLabel="{{'tb.rulenode.drop-file' | translate}}">
+          </tb-file-input>
+          <tb-file-input style="padding-bottom: 8px;"
+                         formControlName="privateKey"
+                         inputId="privateKeySelect"
+                         [existingFileName]="azureIotHubConfigForm.get('credentials').get('privateKeyFileName').value"
+                         (fileNameChanged)="azureIotHubConfigForm.get('credentials').get('privateKeyFileName').setValue($event)"
```

Comment:
Need change `get('credentials').get(‘privateKeyFileName')` to `get(‘credentials.privateKeyFileName')`